packages/context-editor/src/components/AttributePanel.tsx

```diff
@@ -63,17 +65,18 @@ export function AttributePanel({ panelPosition, viewRef }: AttributePanelProps)
 		);
 	};
 	const updateMarkAttr = (index: number, attrKey: string, value: string) => {
-		// console.log(node);
 		const markToReplace = nodeMarks[index];
-		// const newMarks = [...node.marks];
-		// newMarks[index].attrs[attrKey] = value;
-		// setPosition({
-		// 	...position,
-		// 	node: {
-		// 		...node,
-		// 		marks: newMarks
-		// 	},
-		// });
+		const newMarks: Array<Omit<Mark, "attrs"> & { [attr: string]: any }> = [
+			...(node?.marks ?? []),
+		];
+		newMarks[index].attrs[attrKey] = value;
+		setPosition({
+			...position,
+			node: {
+				...node,
+				marks: newMarks as Mark[],
+			},
+		});
```

Comment:
oh thanks for fixing this! I did notice `class` would save sometimes and `id` wouldn't...